Jenkinsfile

```diff
@@ -95,11 +94,12 @@ def mavenBuild(jdk, cmdline) {
       jdk: "$jdk",
       publisherStrategy: 'EXPLICIT',
       globalMavenSettingsConfig: settingsName,
+      options: [junitPublisher(disabled: false)],
```

Comment:
Interesting.  
Do we need to declare the search patterns still?
Or are the `'**/target/surefire-reports/TEST-*.xml,**/target/failsafe-reports/TEST-*.xml'` now default values?

Reply:
`withMaven` is intercepting Maven plugin configuration (see https://github.com/jenkinsci/pipeline-maven-plugin/blob/master/maven-spy/src/main/java/org/jenkinsci/plugins/pipeline/maven/eventspy/handler/SurefireTestExecutionHandler.java )